### PR TITLE
Create elements with contents and children

### PR DESCRIPTION
**Pull request by nwhite89**
Adds on to the current createElement module the possiblity of creating an element with:
- innerHTML (A string representation of contents for an element)
- children (Pre-created HTMLDOMNodes in which to append to the element when created)
- innerElements (Create new HTMLDOMNodes when creating the element and append -- recursively)

### CODE REVIEW

File: tests/create-element.spec.js [hunk 1]
```diff
@@ -45,5 +47,63 @@ define([
...
             expect(counter).toBe(1);
         });
+
+        describe('Creating elements with child elements', function () {
+            var params,
+                ele;
+
+            beforeEach(function () {
+                params = {
+                    innerElements: [
```
Comment (jonman) [inline tests/create-element.spec.js]: Would `nodes` make more sense?

File: src/create-element.js [hunk 2]
```diff
@@ -63,6 +82,26 @@ define([
         attachCss(ele, params.css);
         attachClass(ele, params.cssNames);
 
+        if (params.innerHTML) {
+            innerHTML(ele, params.innerHTML);
+        }
+
+        // Using an object create new elements with this same module
+        if (params.innerElements) {
+            inner = params.innerElements;
```
Comment (jonman) [inline src/create-element.js]: Would `nodes` make more sense?

Comment (nwhite89) [inline src/create-element.js]: Yeah does make sense :+1: will look at changing

Comment (jonman) [inline src/create-element.js]: Thanks

**Pull request merged**